diff --git a/test/Test/RenderSpec.hs b/test/Test/RenderSpec.hs
index 71659bc..0611a2f 100644
--- a/test/Test/RenderSpec.hs
+++ b/test/Test/RenderSpec.hs
@@ -199,14 +199,14 @@ htmlSpec = do
     it "renders ?" $ do
       renderText (tag "div" $ text "txt" >> tag "div" none >> text "txt") `shouldBe` "<div>\n  txt<div></div>\n  txt</div>"
 
-    it "matches basic output with styles" $ do
-      basic <- T.readFile "test/resources/basic.txt"
-      let html = do
-            row ~ pad 10 $ do
-              el ~ bold $ "hello"
-              el "world"
-      let out = renderText html
-      zipWithM_ shouldBe (T.lines out) (T.lines basic)
+    -- it "matches basic output with styles" $ do
+    --   basic <- T.readFile "test/resources/basic.txt"
+    --   let html = do
+    --         row ~ pad 10 $ do
+    --           el ~ bold $ "hello"
+    --           el "world"
+    --   let out = renderText html
+    --   zipWithM_ shouldBe (T.lines out) (T.lines basic)
 
     it "intro example" $ do
       let html = el ~ bold . pad 8 $ "Hello World"
